### ISSUE TITLE
`ValueRange:getValues()` returns null for empty sheet. 

### ISSUE THREAD

**Issue by scyzoryck** [googleapis/google-api-php-client-services#3679]
#### Environment details

  - OS: Linux
  - PHP version: 8.2
  - Package name and version: 

#### Steps to reproduce

  1. Create an empty spreadsheet
  2. Try to fetch data from it.
  3. In result `ValueRange:getValues()` will return `null` what conflicts with the return type hint from the method. 
 
Can we update the return type to null or array, please? :)   

#### Code example

Related to: https://github.com/flow-php/flow/pull/641

**Comment by saranshdhingra** [comment]
So, I discussed this issue internally, and we can't use Union types because we want to support PHP 7.4.
Regarding adding multiple types in the docbloc: These files are auto generated and it might be a while before we add provisions to put multiple return types(_or nullable in this case_) in the generator.

So, this issue might take a while to resolve.

I am sorry I don't come bearing better news, but I just wanted to update you what we discussed.
